### DISCUSSION TITLE
All configurations fail in the first cycle, and max_devi_f reaches the order of 10 to the first power

### DISCUSSION THREAD

**Discussion by 12jscvb** [deepmodeling/dpgen#1376] [General]
After the end of the first cycle of dpgen, all configuration failures occurred, and the reason could not be found. A screenshot of the error and the param.json file are attached below. Thank you for help!
![截图](https://github.com/deepmodeling/dpgen/assets/145745956/4765f190-5bcd-4d1c-850c-08c5d3bc46f9)
![model_devi_f](https://github.com/deepmodeling/dpgen/assets/145745956/b7199f44-74b5-40d4-8d25-3298f98ff2f0)
![param_screeshot](https://github.com/deepmodeling/dpgen/assets/145745956/72172b55-9e18-49c0-b4f4-f629a79fd8ce)

**Comment by njzjz** [comment]
The `max_devi_f` is too large in the first step, which means the training data or the initial structure is completely wrong.

**Comment by 12jscvb** [comment]
I ran the initial dataset through cp2kaimd with a time step of 1fs and ran 200 steps，Is there an error in this step?

**Comment by njzjz** [comment]
You could first check if `dp test` reports the expected RMSE.
https://docs.deepmodeling.com/projects/deepmd/en/master/test/test.html

**Comment by 12jscvb** [comment]
I did not find the expected RMSE in the dp test. What should I do? Additionally, I use relative model error in param.json, f_ Low and f_&nbsp; hi was 0.4 and 0.8, it was found that thirty configurations would be candidates, while the rest were failed configurations



---Original---
From: "Jinzhe ***@***.***&gt;
Date: Sat, Oct 28, 2023 05:02 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [deepmodeling/dpgen] All configurations fail in the first cycle,and max_devi_f reaches the order of 10 to the first power (Issue #1363)




 
You could first check if dp test reports the expected RMSE.
 https://docs.deepmodeling.com/projects/deepmd/en/master/test/test.html
